app/components/UserFeedback/__test__/RatingBox.test.tsx

```diff
@@ -0,0 +1,33 @@
+import { createRemixStub } from "@remix-run/testing";
+import { render } from "@testing-library/react";
+import { FeedbackTranslationContext } from "../FeedbackTranslationContext";
+import { RatingBox } from "../RatingBox";
+
+const YES_RATING = "yes";
+const NO_RATING = "no";
+
+const TRANSLATION_KEY_RECORD = {
+  "yes-rating": YES_RATING,
```

Comment:
also can it be constructed with dynamic object property? 
```suggestion
  [YES_RATING_BUTTON_LABEL_TRANSLATION_KEY]: YES_RATING,
```